app/helpers/form_helper.rb

```diff
@@ -399,4 +399,24 @@ def ie_multipart_fix
       content_tag(:input, '', {:type => "hidden", :name => "_ie_support"})
     end
   end
+
+  def input_group_addon(content, options = {})
+    content_tag :span, class: 'input-group-addon' do
+      content_tag :span, content, options
+    end
+  end
+
+  def datepicker_f(f, attr, options = {})
+    options = {
+      :class => 'bsdatepicker',
```

Comment:
The class is required for the js initialization. If you use `data-provides` it does not translate the datapicker.